### PR TITLE
Add HowLongToBeat Plugin

### PR DESCRIPTION
**Pull request by Tueska**
JavaScript Plugin. Uses HowLongToBeat API wrapper to quickly check their times

### CODE REVIEW

**Comment by jjw24** [comment]
This can now use the new nodejs env setup automation. I will create a merge request to the repo soon.

**Comment by jjw24** [comment]
@Tueska PR ready here https://github.com/Tueska/fl-HowLongToBeat/pull/2 please merge, afterwards I will also merge this in.

File: plugins.json [hunk 1]
```diff
@@ -1296,5 +1296,19 @@
...
+        "Description": "Searches for the provided Name and returns the Playtime from HowLongToBeat",
+        "Author": "Tueska",
+        "Version": "1.0.0",
+        "Language": "executable",
+        "ExecuteFileName": "node.bat",
+        "ActionKeyword": "beat",
+        "Website": "https://github.com/Tueska/fl-howlongtobeat",
+        "IcoPath": "https://github.com/Tueska/fl-HowLongToBeat/blob/master/Images/howlongtobeat.png",
+        "UrlDownload": "https://github.com/Tueska/fl-HowLongToBeat/releases/download/v1.0.0/Tueska.HowLongToBeat.zip",
+        "UrlSourceCode": "https://github.com/Tueska/fl-HowLongToBeat/tree/master"
```
Comment (jjw24) [inline plugins.json:1312]:
```suggestion
        "ID": "dcbb0454f0694431944b1b7f18673a54",
        "Name": "How Long To Beat",
        "Description": "Searches for the provided Name and returns the Playtime from HowLongToBeat",
        "Author": "Tueska",
        "Version": "1.0.0",
        "Language": "JavaScript",
        "ExecuteFileName": "main.js",
        "Website": "https://github.com/Tueska/fl-howlongtobeat",
        "IcoPath": "https://cdn.jsdelivr.net/gh/Tueska/fl-HowLongToBeat/Images/howlongtobeat.png",
        "UrlDownload": "https://github.com/Tueska/fl-HowLongToBeat/releases/download/v1.0.0/Tueska.HowLongToBeat.zip",
        "UrlSourceCode": "https://github.com/Tueska/fl-HowLongToBeat/tree/master"
```

File: plugins.json [hunk 2]
```diff
@@ -1296,5 +1296,18 @@
...
         "DateAdded": "2023-02-11T13:18:58Z",
         "LatestReleaseDate": "2023-02-10T14:14:45Z"
+    },
+    {
+        "ID": "dcbb0454f0694431944b1b7f18673a54",
+        "Name": "How Long To Beat",
+        "Description": "Searches for the provided Name and returns the Playtime from HowLongToBeat",
+        "Author": "Tueska",
+        "Version": "1.0.0",
+        "Language": "JavaScript",
```
Comment (jjw24) [inline plugins.json:1306]:
```suggestion
        "Language": "javascript",
```

**Comment by Tueska** [comment]
> @Tueska PR ready here [Tueska/fl-HowLongToBeat#2](https://github.com/Tueska/fl-HowLongToBeat/pull/2) please merge, afterwards I will also merge this in.

Merged your PR. Thanks a lot, didn’t know it was possible that way!

**Comment by jjw24** [comment]
Yeah feature was released recently, I haven't got around to update the docs yet.

File: plugins.json [hunk 3]
```diff
@@ -1298,6 +1298,19 @@
...
+        "Description": "Searches for the provided Name and returns the Playtime from HowLongToBeat",
+        "Author": "Tueska",
+        "Version": "1.0.0",
+        "Language": "javascript",
+        "ExecuteFileName": "main.js",
+        "Website": "https://github.com/Tueska/fl-howlongtobeat",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/Tueska/fl-HowLongToBeat/Images/howlongtobeat.png",
+        "UrlDownload": "https://github.com/Tueska/fl-HowLongToBeat/releases/download/v1.0.0/Tueska.HowLongToBeat.zip",
+        "UrlSourceCode": "https://github.com/Tueska/fl-HowLongToBeat/tree/master"
+   },
```
Comment (jjw24) [inline plugins.json:1313]:
```suggestion
   },
   {
```

**Review by jjw24 — APPROVED** [review]

**Pull request merged**